.changeset/gentle-badgers-confess.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/react': patch
+---
+
+update types for tokens
```

Comment:
What do you think updating the changeset to be more descriptive? Maybe something like
>Token: Update component type to be `PolymorphicForwardRefComponent`

Reply:
updated!